MesaSuite/Program.cs

```diff
@@ -131,12 +131,12 @@ private static void InitCustomLabelFont()
         {
             try
             {
-                PrivateFontCollection pfc = new PrivateFontCollection();
-                int fontLength = Properties.Resources.MC_FONT.Length;
-                byte[] fontdata = Properties.Resources.MC_FONT;
-                System.IntPtr data = Marshal.AllocCoTaskMem(fontLength);
-                Marshal.Copy(fontdata, 0, data, fontLength);
-                pfc.AddMemoryFont(data, fontLength);
+                //PrivateFontCollection pfc = new PrivateFontCollection();
```

Comment:
This method is effectively doing nothing now, so we can remove it.

Reply:
Talked to Chris, might be best to leave this for now so that we don't need to figure it out all over again in the future.